README.md

```diff
@@ -33,14 +33,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
-
+Semantic HTML is a way to use meaningful element tags that convey information about the content.
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
-
+'display: block;' takes up 100% of available width, and always starts a new line. 'display:inline' Is the opposite; it takes up the minimum width, and does not start a new line.
```

Comment:
Good answer, many people struggle to tell the difference but it seems you understand this concept